libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -122,8 +123,8 @@ void AP_InertialSensor_Backend::_notify_new_gyro_raw_sample(uint8_t instance,
         uint64_t now = AP_HAL::micros64();
         struct log_GYRO pkt = {
             LOG_PACKET_HEADER_INIT((uint8_t)(LOG_GYR1_MSG+instance)),
-            time_us   : now,
-            sample_us : sample_us?sample_us:now,
+            time_us   : sample_us?sample_us:now,
```

Comment:
It's not a strong opinion, but I think that having both time_us and sample_us was better. Just having one doesn't allow to easily know if the sample time is available or not. There's no reason to not just add the sample counter.